### DISCUSSION TITLE
Rad Pro 2.0rc3: Release

### DISCUSSION THREAD

**Discussion by Gissio** [Gissio/radpro#64] [Announcements]
Post your comments here!

**Comment by Montecri** [comment]
Liked the big numbers! Not sure if I understand the bar graph though.

**Reply by DG0MG** [reply]
The bar graph is easy to understand: The longer the bar, the more radiation. :)
And because it doesn't matter whether the difference is 6 or 8 µSv/h, but rather whether it is 8 or 80 or 800 µSv/h, the division is LOGARITHMIC. You can see at first glance what scale you are moving in. There is a factor of 1000 between the “1 µ” and “1 m” labeling!
If you look at older professional devices with a pointer instrument, they often also have a logarithmically divided scale.

**Comment by Gissio** [comment]
It displays the instantaneous rate in a bar graph. This is useful for people who do mineralogy.

**Comment by Montecri** [comment]
Believe 2.0rc3 brought huge improvements. Only have two suggestions:

- Option to simultaneously display counts when any other unit is chosen
- Preserve tube lifetime and life ticks on firmware upgrade.

Thank you Gissio! These devices became much more than what the vendor anticipated. Actually, one of them should sponsor you, I think.

Regards,

- Cristiano

**Reply by Gissio** [reply]
Your first idea is actually fairly easy to implement.

The second idea will be available with the release version.

**Reply by Gissio** [reply]
Could you be more specific regarding the first suggestion? Exactly which units (counts per second, counts per minute, absolute counts) with which view (instantaneous, average, dose)?

**Reply by Montecri** [reply]
Yes. CPM on Instantaneous and Average views.

**Reply by Gissio** [reply]
Montecri: it my understanding that you are a native Portuguese speaker. Could you check the Portuguese translation in 2.2beta3?

The strings are available here: https://github.com/Gissio/radpro/blob/main/platform.io/src/strings/pt.h

If you are willing to help us with these, please follow the discussion here: https://github.com/Gissio/radpro/discussions/241

**Comment by gkulka-web** [comment]
Thank you for big numbers. Great stuff.
